docs/user/dashboard/lens.asciidoc

```diff
@@ -207,7 +211,10 @@ refer to <<explore-fields-in-your-data,Explore the fields in your data>>.
 [[is-it-possible-to-sort-dimensions-in-a-chart]]
 ===== How do I sort the dimensions in a chart?
 
-Sorting dimensions in visualizations is unsupported in *Lens*.
+Sorting dimensions in visualizations is unsupported in *Lens* for charts.
```

Comment:
```suggestion
Sorting chart dimensions is unsupported in *Lens*, but you can sort the dimensions for a single column in data tables.
```

Reply:
I see the copy is shorter with your version, but it does not sound as clear (to me) as having two distinct sentences.